### DISCUSSION TITLE
Beta Projects Navigation is confusing

### DISCUSSION THREAD

**Discussion by michaelsmith-st** [community/community#11280] [Projects and Issues]
I navigated into "Project" wondering what it was, and after a couple interruptions with explanations and "go here to do this" without it actually going there, I finally got to create a new beta project board. 

After I renamed the project and considered what to add to my project...there's no way to get back to the list of projects. It's an orphan view. 

![Screen Shot 2022-02-09 at 8 20 41 PM](https://user-images.githubusercontent.com/80708049/153336575-e80df037-d9ea-479a-91d7-6cb2c2df5608.png)

**Comment by kariem** [comment]
Is this covered in #9597?
